.github/workflows/prerelease.yml

```diff
@@ -11,7 +13,7 @@ env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   GO_VERSION: '1.18'
   INTEGRATION: "oracledb"
-  ORIGINAL_REPO_NAME: 'newrelic/nri-oracledb'
+  ORIGINAL_REPO_NAME: 'nijugeorge173/nri-oracledb'
```

Comment:
🐛

Reply:
Please ignore .. I was trying to setup the go-lint to run for my branch. Its been reverted after test